class.jetpack-cli.php

```diff
@@ -924,6 +924,7 @@ public function sync( $args, $assoc_args ) {
 						'enqueue_wait_time'        => 0,
 						'queue_max_writes_sec'     => 10000,
 						'max_queue_size_full_sync' => 100000,
+						'full_sync_send_duration'  => HOUR_IN_SECONDS,
```

Comment:
😅 Do we really want to do this for an hour? Seems like A LOT!

Reply:
this on CLI, you can Ctrl-C whenever you want :P

Reply:
🌟